docs/en/latest/README.md

```diff
@@ -105,4 +105,4 @@ The recommended approach is to deploy APISIX with [AWS CDK](https://aws.amazon.c
 
 ### Kubernetes
 
-See [this guide](../kubernetes/README.md) and learn how to deploy apisix in Kubernetes.
+See [this guide](../../../kubernetes/README.md) and learn how to deploy apisix in Kubernetes.
```

Comment:
Why not put this file under the current folder?

Reply:
![image](https://user-images.githubusercontent.com/72343596/109408978-94fe1900-79c9-11eb-87b7-63bba3330d71.png)
Can I put that folder into docs, I see that there are other types of files in that folder.

Reply:
You can NOT put a markdown file outside the `latest` folder. Otherwise, there will be an error when building the APISIX website.

Reply:
already removed that README